src/components/chart/boxplot.js

```diff
@@ -0,0 +1,90 @@
+import ChartConstants from './chart-constants';
+
+class Boxplot {
+    constructor() {
+        this.constants = ChartConstants.constants();
+    }
+
+    getBoxplotOptions = (config, useStackedLayout, extraLines) => {
+        return {
+            plotOptions: {
+                boxplot: {
+                    ...this.getPointPadding(config, useStackedLayout, extraLines, false),
+                    boxDashStyle: 'Solid',
+                    fillColor: this.constants.uncertaintyRangeColor,
+                    lineWidth: 0,
+                    medianColor: this.constants.estimateLineColor,
+                    medianDashStyle: 'Solid',
+                    medianWidth: 3,
+                    stemWidth: 0,
+                    whiskerWidth: 0,
+                },
+            },
+            legend: {
+                symbolRadius: 0,
+                itemStyle: {
+                    fontSize: this.constants.defaultFontSize,
+                    color: this.constants.legendLabelColor,
+                },
+            },
+        };
+    };
+
+    getPointPadding = (config, stackedLayout, numberOfExtraLines, isMobile) => {
```

Comment:
I think that currently we are calling this same function from both  `column-chart.js` and `boxplot.js` - perhaps it was already being set in `column-chart.js` when I added my previous comment about this - in which case sorry for confusion.

Reply:
The `column-chart.js` `getPointPadding` function is still getting called twice currently. It's called where you've added it in `boxplot.js` on line 14, but also called where I flagged it in my previous comment - line 211 of `chart.js`